cedar-testing/tests/cedar-policy/main.rs

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
any reason to have `main.rs` instead of `lib.rs`? this is a library crate, right?

Reply:
No special reason, I was just copying the structure of the CLI tests. The files just contain `#[test]`s. Is lib or main more appropriate? Or something else?

Reply:
`lib.rs` is used for crates that provide libraries, `main.rs` is used for crates compiling an executable.  (`main.rs` usually contains a `main()` -- in fact, I'm surprised Cargo isn't complaining that this one doesn't.)  See https://doc.rust-lang.org/cargo/guide/project-layout.html

Reply:
Based on that link, I think that using a `main.rs` file in the `tests` subfolder is the correct structure.

> If a binary, example, bench, or integration test consists of multiple source files, place a main.rs file along with the extra [modules](https://doc.rust-lang.org/stable/cargo/appendix/glossary.html#module) within a subdirectory of the src/bin, examples, benches, or tests directory. The name of the executable will be the directory name.

Also learned: the reason that the `cedar-policy` integration tests weren't running after #707 was the nested directory structure I introduced under `tests/`.